apps/web/package.json

```diff
@@ -44,10 +42,12 @@
     "@vitejs/plugin-react": "4.1.0",
     "autoprefixer": "^10.4.14",
     "postcss": "^8.4.23",
-    "relay-compiler": "^14.1.0",
     "sass": "^1.62.0",
     "typescript": "^4.9.3",
     "vite": "4.5.2",
     "vite-tsconfig-paths": "4.0.5"
+  },
+  "relay": {
+    "artifactDirectory": "src/relay/__generated__"
```

Comment:
To support multiple Relay projects within the same repo/workspaces, I had to add this config here in addition to modifying the `relay.config.json` at the root.